### PR TITLE
Implement merge_equal feature on Policies

### PR DESCRIPTION
**Pull request by leoparente**

### CODE REVIEW

File: src/AbstractMetricsManager.h [hunk 1]
```diff
@@ -519,6 +519,26 @@ class AbstractMetricsManager
         _metric_buckets.at(period)->to_prometheus(out, add_labels);
     }
 
+    void window_external_prometheus(std::stringstream &out, AbstractMetricsBucket *bucket, Metric::LabelMap add_labels = {}) const
+    {
+        if (auto custom = dynamic_cast<MetricsBucketClass *>(bucket); custom) {
+            custom->to_prometheus(out, add_labels);
+        } else {
+            throw std::runtime_error("invalid bucket type");
```
Comment (weyrick) [inline src/AbstractMetricsManager.h]: this could that happen from a policy right, only programmer error?

File: src/Policies.cpp [hunk 2]
```diff
@@ -286,6 +297,10 @@ void Policy::start()
     }
     assert(_input_streams.size());
     spdlog::get("visor")->info("policy [{}]: starting", _name);
+
+    // configurations
+    _merge_equal = (config_exists("merge_equal") && config_get<bool>("merge_equal"));
```
Comment (weyrick) [inline src/Policies.cpp]: how about "merge_like_handlers" instead of "merge_equal"

**Review by weyrick — CHANGES_REQUESTED** [review]
Looks pretty good! Let's get some units and update to the RFC as well.

File: src/AbstractMetricsManager.h (hunk 1 above)
Comment (leoparente) [reply src/AbstractMetricsManager.h]: I've changed that to static and added a comment. It only programmer error

File: src/Policies.cpp (hunk 2 above)
Comment (leoparente) [reply src/Policies.cpp]: cool. changed that

**Review by weyrick — APPROVED** [review]
cool let's try it!

**Pull request merged**